x-pack/plugins/enterprise_search/public/applications/shared/enterprise_search_url/external_url.test.ts

```diff
@@ -28,16 +28,22 @@ describe('Enterprise Search external URL helpers', () => {
     it('generates a public Enterprise Search URL', () => {
       expect(getEnterpriseSearchUrl()).toEqual('http://localhost:3002');
       expect(getEnterpriseSearchUrl('/login')).toEqual('http://localhost:3002/login');
+      expect(getEnterpriseSearchUrl('/login', '/a')).toEqual('http://localhost:3002/a/login');
+      expect(getEnterpriseSearchUrl('/login', '/a', true)).toEqual('http://localhost:3002/a#/login');
     });
 
-    it('generates a public App Search URL', () => {
-      expect(getAppSearchUrl()).toEqual('http://localhost:3002/as');
-      expect(getAppSearchUrl('/path')).toEqual('http://localhost:3002/as/path');
+    describe('App Search URL helper', () => {
```

Comment:
This is not a blocking comment, but just calling it out. Why did you decided to add a `describe` block to the app search helper but not the workplace search helper?

Reply:
I removed it. Initially, I wanted to add a `describe` for the method without anchor and another one for the method with it